### DISCUSSION TITLE
Heist Control 2.29.3

### DISCUSSION THREAD

**Discussion by jhowkNx** [jhowkNx/Heist-Control-v2#53] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/jhowkNx/Heist-Control-v2/releases/tag/2.29.3'>Heist Control 2.29.3</a>.</em>

**Comment by igobythisname** [comment]
just completed quick cayo perico heist and everyone's pay was $2.6m, so no one got paid

**Comment by Tomalisa** [comment]
So I have been using heist control without activating the option all the time. First, I activate a quick setup option, say solo, then I deactivate it, then I go to my submarine and start up the heist so that I get to host it like a normal heist, where I can edit payment, approach, etc. And I and other players can get paid (as long as it didn't exceed the limit), so I wonder if I was using it correctly or not?

**Reply by jhowkNx** [reply]
You should only disable the preset after finishing the heist.
But since you want to change payments yourself, this can be a good one. However, the loots of some presets are calculated according to percentage, changing this to higher or lower in some cases, will cause wrong payments to other players and you as well.

**Reply by Tomalisa** [reply]
For now, it seems like its just like a normal heist (and still, as long as the payout isn't too much), except the only flaw is that it is not always the maximum payout but I don't mind that, free easy money that I don't have to do any preparation afterall.

So another question is, with my way of using the script feature (activate quick setup and press again to immediately deactivate), does the quick setup make any difference? Like solo duo trio 2.4, do they mean the same thing to me?

**Comment by twedyy** [comment]
Hey is this script still works?

**Reply by jhowkNx** [reply]
Working without problems.
